api/src/controllers/events/click-event.ts

```diff
@@ -29,4 +29,22 @@ const clickEventRoute = APIWrapper({
     },
 });
 
-export const clickEvent = relogRequestHandler(clickEventRoute);
\ No newline at end of file
+const clickEventPaginationRoute = APIWrapper({
+    GET: {
+        config: {
+            requireToken: true,
+        },
+        handler: async (req: Request) => {
+            const { afterTime, afterId, limit, projectName } = req.params;
+
+            if (!afterId || !afterTime || !limit || !projectName) {
+                throw new Error("You must specify a project name to create a project!")
```

Comment:
If afterId is undefined/null then it should just get you the first page of events